src/coreclr/jit/likelyclass.cpp

```diff
@@ -0,0 +1,212 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+XX                                                                           XX
+XX                           Likely Class Processing                         XX
+XX                                                                           XX
+XX   Parses Pgo data to find the most likely class in use at a given         XX
+XX   in a method. Used by both the JIT, and by crossgen                      XX
```

Comment:
```suggestion
XX   offset in a method. Used by both the JIT, and by crossgen.             XX
```
maybe?

Reply:
Yes. I'm not sure where the IL offset bit of my comment went... I certainly remember writing it. Thanks for noting this.